examples/app-python/example_app.py

```diff
@@ -19,9 +19,20 @@
 import os
 import sys
 import time
+import serial
 
 from satos_payload_sdk import app_framework
+from satos_payload_sdk import antaris_api_gpio as api_gpio
 
+# Note : Assign proper values of GPIO Pin and Port, as per your hardware interface. 
+g_GPIO_ERROR = -1
+g_GPIO_Port = 0
+g_GPIO_Read_Pin = 5
+g_GPIO_Write_Pin = 6
+
+# Note : Assign proper pin numbers for serial port testing, as per your hardware interface.
+g_Uart_Port = '/dev/ttyUSB0'
+g_Uart_Baudrate = 9600
```

Comment:
It's OK to keep these here to start, but in the future we should map this configuration through the payload sequence parameters.